### ISSUE TITLE
Getting stuck waiting for an answer from Quakenet

### ISSUE THREAD

**Issue by Dune-jr** [irccloud/irccat#15]
The bot works wonders on Freenode (thanks!), but somehow can't work on Quakenet with the same basic configuration.

Also Ctrl+C does not work, it says
```
2019/03/25 12:13:33 --> QUIT :Exiting on interrupt
```
and does not exit

Here is a log:

```
2019-03-25 11:09:16 INFO main main.go:37 IRCCat  () starting...
2019/03/25 12:09:16 Connected to irc.quakenet.org:6667 (83.140.172.212:6667)
2019/03/25 12:09:16 Negotiating caps
2019/03/25 12:09:16 Waiting for all ACKs or NAKs
2019/03/25 12:09:16 0 / 3
2019/03/25 12:09:16 --> CAP LS
2019/03/25 12:09:16 <-- NOTICE AUTH :*** Looking up your hostname
2019/03/25 12:09:16 <-- NOTICE AUTH :*** Checking Ident
2019/03/25 12:09:16 <-- NOTICE AUTH :*** Found your hostname
2019/03/25 12:09:16 <-- NOTICE AUTH :*** No ident response
2019/03/25 12:10:46 <-- ERROR :Closing Link:  by port80c.se.quakenet.org (Registration Timeout)
^C2019-03-25 11:13:33 INFO main main.go:83 Exiting on interrupt
2019/03/25 12:13:33 --> QUIT :Exiting on interrupt
^Z
[12]+  Stopped                 go run *.go
```

I'm not very good with Go, but it apparently gets stuck there:

```go
        for i := 0; i < len(irc.RequestCaps); i++ {
                irc.Log.Printf("%d / %d", i, len(irc.RequestCaps))
                <-cap_chan
        }
```

**Comment by russss** [comment]
This should be fixed in v0.4.2 (via thoj/go-ircevent#124)

**Comment by Dune-jr** [comment]
Thank you!
